bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -696,15 +695,12 @@ PrePrepareMsg *ReplicaImp::createPrePrepareMessage() {
 
   if (config_.timeServiceEnabled) {
```

Comment:
Can we remove this check. Just return the preprepare in both cases.

Reply:
correct